SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1525,6 +1696,9 @@ int32 USpatialNetDriver::ServerReplicateActors(float DeltaSeconds)
 	// Build the consider list (actors that are ready to replicate)
 	ServerReplicateActors_BuildConsiderList(ConsiderList, ServerTickTime);
 
+	TArray<FMigrationInfo> ActorsToMigrate;
+	ServerReplicateActors_HandleLoadBalancing(ConsiderList, ActorsToMigrate);
```

Comment:
This is not hit when using the replication graph

Reply:
Discussed offline, this will be tackled in a subsequent PR.
https://improbableio.atlassian.net/browse/UNR-3790